starter-code/js/index.js

```diff
@@ -2,10 +2,28 @@ function deleteItem(e){
 
 }
 
+//Just getting the item price
 function getPriceByProduct(itemNode){
+  var itemUnit = document.getElementsByClassName('item-price');
+  var itemUnitPrice = itemUnit[0].childNodes[itemNode].innerHTML;
+  return itemUnitPrice
+}
+
 
+document.addEventListener('keyup', getUnitsByProduct)
```

Comment:
I see what are you trying here, but you need to update the DOM inside getUnitsByProduct (or call a function that changes the DOM).